viewing_party/party.py

```diff
@@ -0,0 +1,150 @@
+
+def create_movie(movie_title, genre, rating):
+    movie = {}
+    if movie_title is None or genre is None or rating is None:
+        return None
+    else:
+        movie["title"] = movie_title
+        movie["genre"] = genre
+        movie["rating"] = rating    
+        return movie
```

Comment:
The `else` clause is here can be omitted as well. This is a common pattern called the 'guard clause' where you check for a failed case early and allowing the 'main' logic to be in focus and unindented.

```suggestion
    if movie_title is None or genre is None or rating is None:
        return None

    movie["title"] = movie_title
    movie["genre"] = genre
    movie["rating"] = rating    
    return movie
```